pkgs/tools/misc/ets/default.nix

```diff
@@ -0,0 +1,37 @@
+{ system, lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "ets";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "zmwangx";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-XWIDo5msTMTLr60x1R9cwsiZIDG6G+uHWx8idt4F2iA=";
+  };
+
+  patches = [ ./go-mod.patch ];
```

Comment:
Can you also submit this patch upstream, unless you done so?

Reply:
Done.